pkg/api/unversioned/well_known_taints.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
We already have well-known labels in api/unversioned/well_known_labels.go, so this seems the most natural place to put api/unversioned/well_known_taints.